src/test/resources/org/jenkinsci/plugins/durabletask/CentOSFixture/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM centos:7.4.1708
 RUN yum -y install \
-        openssh-server-7.4p1-13.el7_4 \
-        java-1.8.0-openjdk-headless-1.8.0.151-1.b12.el7_4 \
+        openssh-server \
```

Comment:
@jglick Why?

Reply:
See https://github.com/jenkinsci/durable-task-plugin/pull/72/commits/621cfbef0232e176059e834fa465d8a3fcff8272 and a long series of PRs in `docker-fixtures`.